test/gurka/test_time_tracking.cc

```diff
@@ -218,7 +218,8 @@ TEST(TimeTracking, routes) {
       }
     }
   }
-  std::vector<double> expected = {0, 17.1429, 34.2857, 176.789, 228.2844, 252.2844, 276.2844};
+  std::vector<double> expected = {0,        17.1429,  34.2857,  51.4286,
```

Comment:
these were the ones that changed as a result of fixing the bug in the gurka findEdge function

Reply:
correct.